test/unit/data.readers/process.space.tree.test.ts

```
@@ -0,0 +1,242 @@
+
+    const result = await processSpaceTree(spaces as any, mockAlkemioClient);
+
+    expect(mockGenerateDocument).toHaveBeenCalledTimes(3);
+    expect(result).toHaveLength(3);
+    expect(result[0].metadata.documentId).toBe('root');
+    expect(result[1].metadata.documentId).toBe('child-1');
+    expect(result[2].metadata.documentId).toBe('child-2');
+  });
+
+  it('should handle deeply nested subspaces', async () => {
+    let callCount = 0;
```

Biome (2.4.9)

[error] This variable callCount is unused.

(lint/correctness/noUnusedVariables)